### PR TITLE
MRG: Draw sensors

### LINKED ISSUE #2330: implement draw_sensors
implement draw_sensors

https://github.com/mne-tools/mne-matlab/blob/master/matlab/mne_load_coil_def.m#L160

in plot_trans function.

### PR DESCRIPTION
**Pull request by Eric89GXL**
So far I've modified the example and it looks good.

Before:

before

After:

after

I'm thinking of adding to the example to plot layouts from other systems since we have those now. Think it's worthwhile? I guess I should at least beef up the tests so that our coverage doesn't take a hit.

Closes #2330.

### CODE REVIEW

**Comment by agramfort** [comment]
looks clean to me.

see if you can avoid coverage decrease

thanks heaps @Eric89GXL

**Comment by Eric89GXL** [comment]
Coverage should be fine (it now uses all draw types) and I added a new example. Ready for review/merge from my end.

File: examples/visualization/plot_meg_sensors.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+bti_path = op.abspath(op.dirname(mne.__file__)) + '/io/bti/tests/data/'
+raws = dict(
+    Neuromag=Raw(sample.data_path() + '/MEG/sample/sample_audvis_raw.fif'),
+    CTF_275=read_raw_ctf(spm_face.data_path() +
+                         '/MEG/spm/SPM_CTF_MEG_example_faces1_3D.ds'),
+    Magnes_3600wh=read_raw_bti(bti_path + 'test_pdf_linux',
+                               bti_path + 'test_config_linux',
+                               bti_path + 'test_hs_linux')
+)
```
Comment (kingjr) [inline examples/visualization/plot_meg_sensors.py]: any reason not to put the KIT?

Comment (Eric89GXL) [inline examples/visualization/plot_meg_sensors.py]: Whoops, forgot that one

Comment (agramfort) [inline examples/visualization/plot_meg_sensors.py]: so still WIP?

Comment (Eric89GXL) [inline examples/visualization/plot_meg_sensors.py]: Yeah, I should be able to get to it Monday

**Comment by Eric89GXL** [comment]
Okay, KIT added. Ready for review/merge from my end.

**Comment by agramfort** [comment]
works fine for me.

I am wondering if we could display meg_ref channels in another color than blue? Also dark blue is not super visible on black background

if you feel it should be done is next PR feel free to merge.

**Comment by Eric89GXL** [comment]
Sure, what color would you like?

**Comment by Eric89GXL** [comment]
I'll leave it for another PR :)

**Comment by agramfort** [comment]
thx :)

**Pull request merged**